numpy/testing/utils.py

```diff
@@ -674,33 +685,12 @@ def assert_array_compare(comparison, x, y, err_msg='', verbose=True,
     x = array(x, copy=False, subok=True)
     y = array(y, copy=False, subok=True)
 
-    def safe_comparison(*args, **kwargs):
```

Comment:
Heh, I am wondering if it should be worrisome that only one test needed fixup.... but my brain refuses to really think about it right now ;)